docs/rafter/03-04-minio-gateway.md

```diff
@@ -30,6 +30,8 @@ For the production purposes, Rafter uses MinIO Gateway which:
 
 ![](./assets/minio-gateway.svg)
 
+>**TIP:** Every cloud provider offers a different payment policy for storing buckets. To avoid unexpected costs, verify the payment policy with the given provider before you start using Gateway mode. To reduce the costs in general, always try to limit the number of buckets and create them for groups and domains rather than separate assets.
```

Comment:
This is not always true, for example in GCP you are not paying for buckets, only for stored data and traffic. I would rather say it more generic - using a gateway mode may generate additional costs that depend on the pricing policy of a particular cloud provider.